src/style.css

```diff
@@ -0,0 +1,28 @@
+.starts {
+  display: flex;
+}
+
+.stars__star {
+  width: 50px;
+  height: 50px;
+  background-image: url(images/star.svg);
+  margin: 0;
+  flex: 1;
+}
```

Comment:
1. Star's dimensions should be waaay smaller. Try 16px and put it in a variable since this value will repeat
2. Use `background-repeat: no-repeat;`. Keep in mind that according to the checklist all background properties should be combined using a shorthand.